### DISCUSSION TITLE
having a problem billing the organisation. please update your payment information

### DISCUSSION THREAD

**Discussion by altafshaikh1** [community/community#109305] [New to GitHub]
having a problem billing the organisation. please update your payment information. Unable to do the payment via multiple cards. After saving the card there is no option to pay it. Please help

**Comment by queenofcorgis** [comment]
Hi there @altafshaikh1 - have you followed the steps in [this doc](https://docs.github.com/en/billing/managing-your-github-billing-settings/adding-or-editing-a-payment-method#updating-your-organizations-payment-method)?

**Reply by altafshaikh1** [reply]
yes already did that with two different cards and now i tried with paypal account also
